test/controllers/guests_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GuestsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
```

Comment:
we don't use test directory at all, delete unused

Reply:
Deleted